specifications/terminology.md

```diff
@@ -1,3 +1,17 @@
 # Terminology
 
-The following terms are used to describe concepts in this specification.
\ No newline at end of file
+The following terms are used to describe concepts in this specification.
+
+- <dfn>DID</dfn> - A decentralized identifier as defined by [[[did-core]]].
+- <dfn>Holder</dfn> - An entity that possesses a set of identity resources as defined by [[[vc-data-model]]].
+  The holder will typically be the subject of a [=Verifiable Credential=].
+- <dfn>Resource</dfn> - A resource is an entity managed by the Credential Service such as a [=Verifiable Credential=]
```

Comment:
Maybe the definition of "Credential Service" would also be nice to have here, which I proposed in my PR #91.
```
- **Credential Service** - The CS is a network-accessible service that manages Verifiable Credentials [[vc-data-model]]. This includes read and write endpoints
  for Verifiable Presentations (VPs) and Verifiable Credentials (VCs).
```

Reply:
The CS is defined in the ecosystem section. Let's decide whether to keep it there, merge the sections, or duplicate definitions in a subsequent PR.